### PR TITLE
added sg300 module

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Adds support the Cisco SG300 switch
# Impacts

Switch support
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Added support for Cisco SG300 switches

### CODE REVIEW

File: lib/pf/Switch/Cisco/SG300.pm [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Translate RADIUS NAS-Port into the physical port ifIndex
+Just returns the NAS-Port
+
+=cut
+
+sub NasPortToIfIndex {
+    my ($this, $NAS_port) = @_;
+    my $logger = Log::Log4perl::get_logger(ref($this)); 
+
+    $logger->warn("Found $NAS_port for ifindex");
```
Comment (dwlfrth) [inline lib/pf/Switch/Cisco/SG300.pm]: Shouldn't that be 'DEBUG' ? :nail_care:

File: lib/pf/Switch/Cisco/SG300.pm [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+# CAPABILITIES
+# access technology supported
+# inherited from cisco 2960
+#
+
+=head2 getVoipVsa
+
+Get Voice over IP RADIUS Vendor Specific Attribute (VSA).
+For now it returns the voiceVlan untagged since Juniper supports multiple untagged VLAN in the same interface
```
Comment (dwlfrth) [inline lib/pf/Switch/Cisco/SG300.pm]: Good for Juniper, what about Cisco ? ;)

**Comment by julsemaan** [comment]
re-review

**Comment by dwlfrth** [comment]
:rocket:

**Pull request merged**